package.json

```diff
@@ -160,6 +160,7 @@
     "react-chartjs-2": "=2.8.0",
```

Comment:
NRTTL
How do i test this? Where can i download a json to test with?

Reply:
You can get a JSON by backing up your account

Reply:
I only see PDF option 
![image](https://github.com/LiskHQ/lisk-desktop/assets/8784876/0e57f15e-5c1a-43b3-92e7-dba7bd2652c9)

Reply:
continue by clicking on the `I have written them down` then a modal should be seen prompting you to download the encrypted-account  json file @oskarleonard